keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/ResourceTagger.kt

```diff
@@ -83,25 +83,53 @@ class ResourceTagger(
 
   @EventListener(ResourceCreated::class)
   fun onCreateEvent(event: ResourceCreated) {
-    if (event.resourceId.shouldTag()) {
+    if (!event.resourceId.shouldTag()) {
+      return
+    }
+    /**
+     * If a MultiRegion resource provides a set of regional resourceIds, entity tags will be created
+     * against those ids only, not the
+     */
+    if (event.resourceIds.isEmpty()) {
       log.debug("Persisting tag desired for resource {} because it exists now", event.resourceId)
       val spec = event.resourceId.generateKeelTagSpec()
```

Comment:
Does this need to have its own id passed into it now? Or maybe I'm confused...

Reply:
It doesn't, the unparameterized `generateKeelTagSpec()` is still present (now on line 192) and refers to its own id. It just doesn't appear in the diff.